core/mono-samples/iOS/Program.csproj

```diff
@@ -0,0 +1,55 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+    <RuntimeIdentifier>ios-x64</RuntimeIdentifier>
+    <OutputType>Exe</OutputType>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Runtime.iOS.Sample.Mono" Version="6.0.0-dev" />
```

Comment:
I think this won't work with the final nuget package since it won't have a `6.0.0-dev` version like your local build but a real version like `6.0.0-alpha.1.20520.2`.

So what we can do is either hardcode a version or use `Version="6.0.0-*`.

Reply:
Thanks!